packages/grafana-ui/src/themes/dark.ts

```diff
@@ -55,7 +55,7 @@ const textColors = {
   text: basicColors.gray85,
   textSemiWeak: basicColors.gray70,
   textWeak: basicColors.gray60,
-  textFaint: basicColors.gray33,
+  textFaint: basicColors.gray53,
```

Comment:
Not sure this is the best approach since modifying colors in the grafana-ui like this will have impact on lots of other places apart from search.